test/plugin.js

```diff
@@ -47,6 +47,21 @@ describe('Plugin', function () {
     });
   });
 
+  it('should support webpack config with interpolation in `output.path`', async function () {
+    const config = makeWebpackConfig();
+    config.output.path += '/[hash]';
+
+    await webpackCompile(config);
+
+    const chartData = await getChartDataFromReport('7e56a1f8a54acedcc9ce/report.html');
```

Comment:
I'm not sure if this hash is good across different webpack versions... this might cause a very brittle test.

I don't yet have good ideas on how to figure out the hash that was used to get to the report page.